webplugin/js/app/sentry-8.39.0.js

```diff
@@ -0,0 +1,331 @@
+const KM_SENTRY_CONFIG = {
```

Comment:
is this sentry's library file code present in this file, or our written code?

Reply:
We write this config, but the sentry provides the script itself.
Why did we add the script method? 
because, in new sentry packages, they have two methods 

- one using the script(like we provide to our customer) 
- Using the npm package 

The sentry npm packages are modules based which is not supported in web SDK that's why we chose the script method